pytrip/dicomhelper.py

```diff
@@ -19,11 +19,9 @@
 """
 Auxiliary functions for handling Dicom data.
 """
-import os
-try:
-    import pydicom as dicom  # as of version 1.0 pydicom package should be used this way
-except ImportError:
-    import dicom  # fallback to old (<1.0) pydicom package version
+
+import pydicom as pd
```

Comment:
That is how we usually import pandas (`import pandas as pd`).

I would suggest to follow pydicom documentation (https://pydicom.github.io/pydicom/stable/guides/user/base_element.html)  and simply do:

```python
from pydicom import dcmread
```